releasenotes/notes/1.3/prepare-1.3.0-7c45598775fc2bbf.yaml

```diff
@@ -0,0 +1,22 @@
+---
+prelude: >
+    Qiskit version 1.3.0 brings in major performance and quality improvements
+    for the transpiler. The minimum required Python version is now 3.9.
+
+    There have been many new features, fixes and improvements introduced in this
+    new version of Qiskit, some of which include:
+
+      * The majority of the transpiler passes used by the preset pass 
+        manager have been ported into Rust, resulting in a 6x overall runtime 
+        improvement compared to Qiskit 1.2.4. The ported passes include :class:`.BasisTranslator`,
+        :class:`.CommutationAnalysis`, :class:`.ConsolidateBlocks`, :class:`.UnitarySynthesis`,
+        and others.
+
+      * Improvements to the circuit library that increase compilation quality and
```

Comment:
Do we want to call out ancilla allocation in hls here?

Reply:
How can I better write something for this? @alexanderivrii

Reply:
Just yesterday @Cryoris and I were writing a paragraph for Luciano's blog post, so trying to shorten what we wrote and combine with what you already have, I can suggest writing something like this:

```
* Improvements to the circuit library that increase compilation quality and
  circuit construction speed. Structural operations that have a unique
  decomposition are represented as functions that return a :class:`.QuantumCircuit`. 
  New examples include :func:`.quantum_volume`, :func:`.qaoa_ansatz`, and 
  :func:`.efficient_su2`. Most of these functions are built in Rust, resulting in 
  significantly faster circuit construction runtime.
  The abstract operations that can be implemented in multiple different ways
  in terms of the lower-level gates are represented as :class:`.Gate` or 
  :class:`.Instruction`. New examples include :class:`.ModularAdderGate`, 
  :class:`.MCMTGate`, and :class:`.GraphStateGate`. This allows building a quantum 
  circuit without choosing a specific implementation for these gates, and letting 
  the compiler choose the optimal decomposition. This is especially powerful
  in combination with the improvements to the :class:`.HighLevelSynthesis`
  transpiler pass that tries to find the highest-quality decomposition available
  for a given gate, and can now leverage idle auxiliary qubits. There is a large
  number of new available plugins, including a new plugin for 
  :class:`.PauliEvolutionGate` that leverages the circuit synthesis library 
  `Rustiq <https://github.com/smartiel/rustiq>`_.
```
But you may want to shorten this a bit further and improve the wording.

Reply:
Extending on Sasha's suggestion:
```
* Improvements to the circuit library that increase compilation quality and
  circuit construction speed. Operations are now distinguished into:
  
   * Structural operations, that have a unique decomposition, and are 
      represented as functions that return a :class:`.QuantumCircuit`
      (for example: :func:`.real_amplitudes`). 
      Most of these functions are built in Rust, resulting in 
      significantly faster circuit construction runtime.
   * Abstract operations, that can be implemented using different 
      decompositions, are represented as :class:`.Gate` or :class:`.Instruction`
      (for example: :class:`.PauliEvolutionGate`).
      This allows building an abstract quantum circuit and letting 
      the compiler choose the optimal decomposition.
      
  Using an abstract circuit description is especially powerful in combination 
  with improvements to the :class:`.HighLevelSynthesis` transpiler pass,
  which can now take into account idle auxiliary qubits to find the best available 
  decomposition for a given gate.
```